.github/workflows/bcny-firebase.yml

```diff
@@ -52,3 +52,37 @@ jobs:
         with:
           name: firebase-windows-amd64
           path: ${{ github.workspace }}/BuildRoot/Library/firebase
+
+      - run: |
+          echo build_tag=$(date +%Y%m%d.$(date +%-H/6 | bc)) >> ${GITHUB_ENV}
```

Comment:
lol what's up w/ the inner `$()`?
(why is quarter-day a meaningful unit here?)

Reply:
Quarter day is a meaningful unit only in that it is copy-pasta.  We really just need a counter.  The toolchain builds build occur every 6 hours, so we get quarter days.

Reply:
lol IMO just use full hours.minutes.seconds and avoid potential future confusion. But that's just me.